proms-resources/frontend/src/main/frontend/src/proms/SurveyInstConfiguration.jsx

```diff
@@ -0,0 +1,165 @@
+//
+//  Licensed to the Apache Software Foundation (ASF) under one
+//  or more contributor license agreements.  See the NOTICE file
+//  distributed with this work for additional information
+//  regarding copyright ownership.  The ASF licenses this file
+//  to you under the Apache License, Version 2.0 (the
+//  "License"); you may not use this file except in compliance
+//  with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing,
+//  software distributed under the License is distributed on an
+//  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//  KIND, either express or implied.  See the License for the
+//  specific language governing permissions and limitations
+//  under the License.
+//
+import React, { useEffect, useState } from 'react';
+import {
+    List,
+    ListItem,
+    TextField,
+    Typography
+} from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import AdminConfigScreen from "../adminDashboard/AdminConfigScreen.jsx";
+
+const useStyles = makeStyles(theme => ({
+  header: {
+    marginTop: theme.spacing(3),
+  },
+}));
+
+function SurveyInstConfiguration() {
+  const classes = useStyles();
+
+  const [ surveyInstructions, setSurveyInstructions ] = useState();
+  const [ hasChanges, setHasChanges ] = useState(false);
+
+  const defaults = {
+    noSurveysMessage: "You have no pending surveys to fill out"
+  };
```

Comment:
It would be nice to use the defaults as placeholders for the text fields. They can be defined & exported in this file, displayed as placeholders when applicable, and imported by QuestionnaireSet to be used in case there are fields not defined by the user.

```suggestion
 export const defaultInstructions = {
    noSurveysMessage: "You have no pending surveys to fill out"
  };
```